changelog.mdx

```diff
@@ -7,6 +7,56 @@ description: Updates to Chalk!
 
 ---
 
+## September 23, 2024
+
+### Configurable retry policy for SQL resolvers
+
+Sometimes, a SQL resolver may fail to retrieve data due to temporary unavailability. We've added new options for
+configuring the number of retry attempts a resolver may make (and how long it should wait between attempts). If you're
+interested in trying out this new functionality early, please let the team know.
+
+### Has-one join keys can be chained
+
+When creating [has-one](/docs/has-one) relationships, you can set the primary key of the child feature class to the
+primary key of the parent feature class. For example, you may model an `InsurancePolicy` feature class as belonging to
+exactly one user by setting `InsurancePolicy.id`s type to `Primary[User.id]`.
+
+Now, we've updated Chalk so that you can chain more of these relationships together. For example, a `InsurancePolicy`
+feature class may have an associated `InsuranceApplication`. The `InsuranceApplication` may also have an associated
+`CreditReport`. Chalk now allows chaining an arbitrary number of has-one relationships. Chalk will also validate these
+relationships to ensure there are no cyclical dependencies.
```

Comment:
"cyclical dependencies" vaguely caught my eye, so I tried to look up whether it should be "cyclic dependencies" which felt right to me, but it's not clear if one is more or less correct. I see more references to "cyclic" than "cyclical" but I don't think it really matters

Reply:
modified to "circular" (later commit)